website/docs/d/cbr_rule.html.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_cbr_rule"
+description: |-
+Get information about cbr_rule
+subcategory: "Context Based Restrictions"
+---
+
+# ibm_cbr_rule
+
+Provides a read-only data source for cbr_rule. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_cbr_rule" "cbr_rule" {
+	rule_id = "rule_id"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+* `rule_id` - (Required, Forces new resource, String) The ID of a rule.
```

Comment:
Please remove `Forces new resource`, datasource ll not have any force new resource

Reply:
removed Forces new resource